rpc/api/api.proto

```diff
@@ -0,0 +1,114 @@
+syntax = "proto3";
+
+import "google/api/annotations.proto";
+
+package api;
+
+service Post {
```

Comment:
What about a validate call? (for testing purposes)

Reply:
I thought it should be done by consuming the package directly, and not via the RPC server (which is basically the prover side). But we can add it to the RPC to easily support non-Go clients. 

And this is how we're testing now - proving via RPC server, and validating by using the package directly.